hooks/useAuctionTopBid/useAuctionTopBid.ts

```diff
@@ -0,0 +1,45 @@
+import { useConfig } from 'hooks/useConfig';
+import { useOracleInfo } from 'hooks/useOracleInfo/useOracleInfo';
+import {
+  getTopBidForCollectionAtTime,
+  OraclePriceType,
+} from 'lib/oracle/reservoir';
+import { useEffect, useMemo, useState } from 'react';
+import { AuctionQuery } from 'types/generated/graphql/inKindSubgraph';
+
+export function useAuctionTopBid(
+  auction: NonNullable<AuctionQuery['auction']>,
+) {
+  const config = useConfig();
+  const oracleInfo = useOracleInfo(OraclePriceType.spot);
+  const [topBidAtEnd, setTopBidAtEnd] = useState<number | null>(0);
```

Comment:
should the default case for this be 0 or null?

Reply:
null, thanks